pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -19,17 +15,24 @@ social:
 %div{dir: language_dir_class}
 
   %h1= I18n.t(:hoc_overview_title)
+  %aside.hoc-skinny-banner
+    .wrapper
+      %h1=hoc_s(:hoc2022_codeorg_title)
+      %p=hoc_s(:hoc2022_codeorg_description)
+    %img{src: "/images/hoc2022-skinny-banner-james-webb-illustration.png", alt: "Illustration of NASA's James Webb telescope"}
```

Comment:
Maybe not something to handle in this PR but is it possible to get the alt text in the i18n pipeline?

Reply:
Yes definitely, good catch! I can add it right now.